[DO NOT MERGE] Test updating classic dark default theme to use "blueberry blue" colors

agrullon95 opened the pull request:
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to https://github.com/Automattic/dotcom-forge/issues/4615

## Proposed Changes

* Testing PR for pink to "blueberry blue" color changes. 
Partially updates the classic-dark theme to match the color scheme and then updates some of the onboarding flows to use blueberry. 

Work in progress for follow up: Other areas such as input fields, hover/focus colors, site icon picker, etc. should ideally be updated as well

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Checkout branch or use live link
* Navigate through main onboarding flows and look at CTA color changes. 
such as `/setup/blog`, `/setup/newsletter`, `/setup/link-in-bio`,
* Navigate through `/log-in`
* Navigate through sign up `/start` then `Continue with Email`

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

agrullon95 committed:
Update classic dark default theme to use studio-blue instead of studio-pink

agrullon95 committed:
Testing changes for classic dark theme color scheme pink to blueberry blue

agrullon95 committed:
Merge branch 'trunk' into update/default-classic-dark-theme-pink-color-scheme-v2

agrullon95 committed:
Merge branch 'trunk' into update/default-classic-dark-theme-pink-color-scheme-v2

agrullon95 committed:
Update adding is-classic-dark class

agrullon95 committed:
Update CTA color in color-scheme svg

agrullon95 committed:
Remove comments

lsl commented:
Not sure about the approach, lets keep discussing on slack.

I don't want this to turn into a huge audit of all the overrides but these are ones I see quickly that might be visible enough to fix up front.

Menus: Selected menu color + hover text color + comment count color
Page: link color should probably update to blueberry?
Zendesk chat color: might not be able to change this but if we can that would be nice to move to blue berry too

![Screenshot_2023-12-14_09-23-05](https://github.com/Automattic/wp-calypso/assets/811776/25cd44f8-b891-49fe-8a55-4f67d42ce58b)

Not sure how to fix this one (if you even should?), might need to change colors for personal, premium and business all at once:
![Screenshot 2023-12-14 at 09-23-36 Plans ‹ test — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/06abc538-c85d-47db-a283-08d4d1db83fb)

agrullon95 committed:
Update classic dark color scheme color-link variable

agrullon95 committed:
Stepper css changes to use blueberry color

agrullon95 committed:
More styling changes across calypso to use theme color accent/blueberry color

agrullon95 committed:
Remove setting is-classic-dark class on document body

agrullon95 committed:
Add gutenberg blueberry color scheme to classic-dark theme file

agrullon95 committed:
Add is-classic-dark class to body to retrieve correct colors on pages

lsl commented on client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss:
```diff
@@ -104,14 +104,14 @@
+				background: var(--color-accent-60);
 			}
 			&:hover {
 				background-color: var(--wp-admin-theme-color-darker-10);
```
Will want a --color-accent-60 here too, (I just replaced it with blue 60 a few minutes ago https://github.com/Automattic/wp-calypso/pull/85337)

agrullon95 replied:
Updated

lsl commented on client/my-sites/plugins/education-footer/index.tsx:
```diff
@@ -210,7 +210,7 @@ function ReadMoreLink() {
 
 	return (
-		<CardText color="var(--studio-blue-50)">
+		<CardText color="#3858e9">
```
Is there somewhere we can put these as variables?

agrullon95 replied:
Hmmm i can look into this , possibly follow up PR

lsl commented:
@agrullon95 could you please pick these blues up too? (either this PR or a follow up :pray: ) Looks like there are a couple more usages of the same hexcode listed under something called "richblue".

https://github.com/Automattic/wp-calypso/blob/59e6b2646cd87da88de99576b35b9ae6971e6b9b/client/login/magic-login/style.scss#L207-L209

Seen here:

https://github.com/Automattic/wp-calypso/assets/811776/c2aaa02e-6e97-4233-b8bb-00ae288cc53f

agrullon95 committed:
Merge branch 'trunk' into update/default-classic-dark-theme-pink-color-scheme-v2

agrullon95 committed:
Updated blog intent button hover

agrullon95 committed:
Update magic login button color and hover

agrullon95 committed:
Update theme colors and hover

agrullon95 commented:
>@agrullon95 could you please pick these blues up too? (either this PR or a follow up 🙏 ) Looks like there are a couple more usages of the same hexcode listed under something called "richblue".

@lsl - I updated the styling for the magic login button and added a hover state since I noticed it was missing on prod.
I think the richblue is fine as is, but i'll look a bit more into it. Thanks for the reviews thus far.

agrullon95 committed:
Update more studio blue to blueberry

lsl commented:
No problem, I'm just glad someone is able to dedicate some time for this.

Have you checked the editor? :smile: 

I'm assuming this is done via etk, but I couldn't get it to apply the new theme using:

```
install-plugin.sh editing-toolkit update/default-classic-dark-theme-pink-color-scheme-v2
```

Maybe I'm not sandboxing the right urls?

agrullon95 committed:
Merge branch 'trunk' into update/default-classic-dark-theme-pink-color-scheme-v2
